backend/src/prompts/templates/intent.j2

```diff
@@ -7,10 +7,10 @@ The task is to comprehend the intention the question. Based on this understandin
 
 Specify an operation type under the operation key; here are a few examples:
 * "literal search" - This should be used when the user is looking to find precise information, such as known facts
-* "relevancy search" - This should be used when the user is looking to find something that is not a literal and is fuzzy - default sort_order relevant descending
-* "filter + aggregation" - This should be used when they want something like a count, where there will be only 1 number returned - default sort_order descending
-* "filter + aggregation + sort" - This should be used when multiple numbers will be returned - default sort_order descending
-* "filter + sort" - This should be used when no aggregation is required e.g. count - default sort_order descending
+* "relevancy search" - This should be used when the user is looking to find something that is not a literal and is fuzzy - default sort_order relevant  descending if income, ascending if expense

```

Comment:
What default sort order will be applied if the user is looking for something that isnt numeric e.g. "What are the titles of all the science fiction movies purchased on netfix last year"

Reply:
It is a good point and the fact that this is just talking about income and expenses means to that it isnt general enough. I shall remove it